.gitignore

```diff
@@ -12,6 +12,9 @@ pids
 *.seed
 *.pid.lock
 
+src/components/icons/raw-components
+src/components/icons/components
```

Comment:
Sorry to be picky, can you move those at the end of the file? Just so that we keep the default grouping